src/UI/Components/Diagram/init.ts

```diff
@@ -212,10 +212,10 @@ export default function diagramInit(
         );
 
         //if there is corresponding rule we can apply connection and update given service
-        if (cellConnectionRule) {
+        if (cellConnectionRule && cellConnectionRule.attributeName) {
```

Comment:
I remember the comment about the Typing and how you distinguished between one sort of Entity and another. I still don't agree with using this to know it's an interservice relation.